switcheroo/ssh/scripts/publish.py

```diff
@@ -66,26 +74,56 @@ def create_argument_parser() -> ArgumentParser:
     return argument_parser
 
 
-def main():
+def _local_store(sshdir: str, bucket: str | None = None) -> FileKeyPublisher:
+    if bucket is not None:
+        raise InvalidArgumentError(
+            'Invalid argument "--bucket" when storing the keys locally'
+        )
+    return FileKeyPublisher(Path(sshdir))
+
+
+def _s3_store(sshdir: str, bucket: str | None = None) -> S3KeyPublisher:
+    if bucket is None:
+        raise MissingArgumentError("The s3 option requires a bucket name!")
+    return S3KeyPublisher(bucket, root_ssh_dir=Path(sshdir))
+
+
+def _metrics(
+    metricpath: str | None = None, metric: str | None = None
+) -> MetricPublisher:
+    if metric == "file":  # publish to file system
+        return FileMetricPublisher(Path(metricpath))
```

Comment:
This doesn't take into account if the metric path is not provided. I made on the main branch an app data dir constant .switcheroo. - for now in this PR we should probably have it hardcoded as .switcheroo/metrics and we can replace .switcheroo with the metric constant on merge

Reply:
If the user choses to store metrics locally and he doesn't specify a path we make a default for them.